.github/workflows/dependency-review.yaml

```diff
@@ -10,3 +10,5 @@ jobs:
     permissions:
       contents: read
       pull-requests: write
+    with:
+      base-ref: github.event.pull_request.base.sha || 'master'
```

Comment:
```suggestion
      base-ref: ${{ github.event.pull_request.base.sha }} || 'master'
```

Sorry, typo.

Reply:
I think it should be `${{ github.event.pull_request.base.sha || 'master' }}`.  I was just about to push but noticed your change.